django/core/cache/backends/base.py

```diff
@@ -115,34 +117,54 @@ def add(self, key, value, timeout=DEFAULT_TIMEOUT, version=None):
         """
         raise NotImplementedError('subclasses of BaseCache must provide an add() method')
 
+    async def aadd(self, key, value, timeout=DEFAULT_TIMEOUT, version=None):
+        """See add()."""
```

Comment:
I don't see much value in `See method().` docstrings when it's only `sync_to_async()` call. Please remove it from `aadd`, `aget`, `aset`, `atouch`, `adelete`, `ahas_key`, `adecr`, `aclear`, and `aclose`.